sdk/storage/azure-storage-blob/azure/storage/blob/_download.py

```diff
@@ -645,10 +646,11 @@ def read(self, size: int = -1, *, chars: Optional[int] = None) -> T:
                 (self._download_complete and self._current_content_offset >= len(self._current_content))):
             return b'' if not self._encoding else ''
```

Comment:
azure\storage\blob\_download.py:647: error: Incompatible return value type (got "Sequence[object]", expected "bytes")  [return-value]
azure\storage\blob\_download.py:647: error: Incompatible return value type (got "Sequence[object]", expected "str")  [return-value]

Thanks to `T` 😄  authorization to use a type-ignore [return-value] ?